commands-yml/commands/example.yml

```diff
@@ -31,8 +31,11 @@ example_usage:
       // CSharp Code here
 
 client_docs:
-  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#click--"
-  python: "https://github.com/appium/python-client"
+  java:
+    - url: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#click--"
+  python:
+    - android: "https://github.com/appium/python-client"
+      ios: "https://github.com/appium/python-client"
```

Comment:
does this line need a `-`? the recording screen one has a `-` for each platform

Reply:
also, both lines are the same...

Reply:
It works either way. If it is under a single dash, then the generating function is called with an array of one element with two properties, and if it is with two dashes it is called with an array of two elements each with one property. Either way it comes out the same.

As for this being the same, this is just the example. But I've changed them.